src/interfaces.js

```diff
@@ -83,9 +83,15 @@ export function mapTagOperations({spec, cb = nullFn, defaultTag = 'default'}) {
       // Id already exists?
       if (typeof tagObj[id] !== 'undefined') {
         // Bump counter ( for this operationId )
-        operationIdCounter[id] = (operationIdCounter[id] || 0) + 1
+        let originalCounterValue = (operationIdCounter[id] || 1)
```

Comment:
Note that the test also makes sure that the tags are in order and not being inadvertently overwritten, by use of the `count` variable in the test.